packages/jest-cli/src/reporters/DefaultTestReporter.js

```diff
@@ -139,6 +140,37 @@ class DefaultTestReporter {
         `${PENDING_COLOR(`${print('test', pendingTests)} skipped`)}, `;
     }
 
+    let snapshotsAdded = 0;
+    let snapshotsFiles = 0;
+    let snapshotsMatched = 0;
+    let snapshotsRemoved = 0;
+    aggregatedResults.testResults.forEach(result => {
+      if (result.snapshotsAdded) {
+        snapshotsFiles++;
+      }
+      snapshotsAdded += result.snapshotsAdded;
+      snapshotsMatched += result.snapshotsMatched;
+      snapshotsRemoved += result.snapshotsRemoved;
+    });
+    if (snapshotsAdded || snapshotsMatched) {
```

Comment:
As you said currently if a snapshot is removed another is added, so this will currently work fine;
Once I implement validation for removed tests this will change, probably better to add it right now, thanks to pointing it out.